Source/ACE/Entity/Player.cs

```diff
@@ -2211,114 +2417,109 @@ public void UpdateAppearance(Container container, uint? itemId)
             }
         }
 
-        public void HandleActionWieldItem(Container container, uint itemId, uint location)
+        /// <summary>
+        /// This method sets properties needed for items that will be child items.
+        /// Items here are only items equipped in the hands.  This deals with the orientation
+        /// and positioning for visual apperence of the child items held by the parent. Og II
+        /// </summary>
+        /// <param name="container"></param>
```

Comment:
please remove the param tags if they're just empty

Reply:
opps - missed those all filled in now.